src/Magnum/DartIntegration/Object.h

```diff
@@ -114,11 +140,36 @@ class MAGNUM_DARTINTEGRATION_EXPORT DartObject: public SceneGraph::AbstractBasic
         dart::dynamics::BodyNode* bodyNode() { return _body; }
 
     private:
-        explicit DartObject(SceneGraph::AbstractBasicObject3D<Float>& object, SceneGraph::AbstractBasicTranslationRotation3D<Float>& transformation, dart::dynamics::ShapeNode* node, dart::dynamics::BodyNode* body);
+        explicit Object(SceneGraph::AbstractBasicObject3D<Float>& object, SceneGraph::AbstractBasicTranslationRotation3D<Float>& transformation, dart::dynamics::ShapeNode* node, dart::dynamics::BodyNode* body);
+
+        /**
+        @brief Convert `ShapeNode` to mesh and material data
+
+        Returns @ref Corrade::Containers::NullOpt if the shape of given `ShapeNode` is
+        not supported. The following DART shapes are supported:
+
+        -   `BoxShape`
+        -   `CapsuleShape`
+        -   `CylinderShape`
+        -   `EllipsoidShape`
+        -   `MeshShape`
+        -   `SoftMeshShape`
+        -   `SphereShape`
+
+        The following DART shapes are not yet supported:
+
+        -   `ConeShape`
+        -   `LineSegmentShape`
+        -   `MultiSphereConvexHullShape`
+        -   `PlaneShape` (this is an infinite plane with normal)
+        */
+        bool _convertShapeNode();
```

Comment:
This is a `private` function and so the documentation will get completely ignored. Can you put that to the class docs instead?

Also, please don't prefix *functions* with underscore. The func is private already, that's enough :)

Reply:
> This is a private function and so the documentation will get completely ignored. Can you put that to the class docs instead?

Of course...